index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+    </div>
+
+    <!-- Headline Section-->
+    <div class="headline">
+
+        <div class="container">
+            <h1>Hey there. I'm Cristian<br><span style="color: #36B37E"> I'm a software developer</span> </h1>
+        </div>
```

Comment:
- [ ] Kindly 😃the text ```I'm a software developer``` try to put them at heading tag less than ```h1 ``` tag because The subtitle (if existing) uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`.

* This way will be semantically correct and the screen readers and other technologies will thank you.💯

| Text |
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/219174293-f64f753f-45b1-4c2e-881d-9b9c485af7af.png) |